tools/mason/MasonEnv.chpl

```diff
@@ -43,6 +43,15 @@ proc MASON_CACHED_REGISTRY {
   return cachedRegistry;
 }
 
+proc MASON_OFFLINE {
```

Comment:
Let's return a `bool` here instead of a string -- that should make it easier to work with elsewhere, e.g. ` if MASON_OFFLINE == 'true' { .. }` can become `if MASON_OFFLINE { .. }`